schema/cassandra.cql

```diff
@@ -0,0 +1,38 @@
+DROP KEYSPACE IF EXISTS gallery_service;
+
+CREATE KEYSPACE gallery_service WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 } AND durable_writes = 'true';
+
+USE gallery_service;
+
+CREATE TABLE IF NOT EXISTS listings (
+  listing_id uuid,
+  title text,
+  listing_name text,
+  rating int,
+  num_reviews int,
+  superhost boolean,
+  listing_location text,
+  hostname text,
+  host_avatar_url text,
+  share boolean,
+  save_feature boolean,
+  PRIMARY KEY(listing_id)
+)
+
+CREATE TABLE IF NOT EXISTS feature_photos (
```

Comment:
This schema is using featured photo id to find listing id. Is this what you meant to do? Because it doesn't quite make sense to me from a use perspective. It makes more sense for me to find the featured photo ids for all the listings.